pkgs/development/python-modules/readchar/default.nix

```diff
@@ -26,11 +23,12 @@ buildPythonPackage rec {
   postPatch = ''
     substituteInPlace setup.cfg \
       --replace "--cov=readchar" ""
+    # run Linux tests on Darwin as well
+    substituteInPlace tests/linux/conftest.py \
+      --replace 'sys.platform.startswith("linux")' 'sys.platform.startswith(("darwin", "linux"))'
```

Comment:
Upstream doesn't accept this change as a PR: https://github.com/magmax/python-readchar/pull/99

Reply:
Maybe add the reference as a comment as well.